mixins/visual/responsive-visual.vue

```diff
@@ -33,6 +56,7 @@ export default
 		@isLanscapeMediaQuery?.removeListener @checkIsLandscape
 
 	computed:
+		isResponsiveImage: -> !!@landscape?.props?.image and !!@portrait?.props?.image and @preload
```

Comment:
Is the `and @preload` required here?  I would think, based on the name, only the checks for landscape and portrait images would be required.